### PR TITLE
add /limit /limit/set to mqtt

### PR DESCRIPTION
**Pull request by KipK**

### CODE REVIEW

File: docs/mqtt.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ Scheduler data:
 `<base-topic>/scheduler/set <json data>`	: set/update schedules ( data as refered from API: https://openevse.stoplight.io/docs/openevse-wifi-v4/e87e6f3f90787-batch-update-schedule )  
 `<base-topic>/scheduler/clear <id>          :`remove related event  
 
+Limit:
+`<base-topic>/scheduler/`    				: get scheduler data ([json data]  
+`<base-topic>/scheduler/set <json data>`	: set/update schedules ( data as refered from API:https://openevse.stoplight.io/docs/openevse-wifi-v4/c410fb5e48294-set-charge-limit )
+`<base-topic>/scheduler/set clear`	: clear current limit
+
```
Comment (jeremypoulter) [inline docs/mqtt.md:51]: Needs updating for limit

Comment (KipK) [reply docs/mqtt.md]: oups, got lost in rebasing. done

**Comment by mathieucarbou** [comment]
This PR will create conflicts with #634: is it possible to merge this one so that I rebase #634 just after ?

**Review by jeremypoulter — APPROVED** [review]

**Comment by jeremypoulter** [comment]
@KipK I have approved, merge when ready

**Comment by mathieucarbou** [comment]
FYI I've tested that in my MQTT HA integration (https://gist.github.com/mathieucarbou/92a3d5e0dc38d6b68aa1bdaf153a80c50) and I can set and clear limits. I have only tested with the `soc` type.

**Pull request merged**